### DISCUSSION TITLE
Citation and reproducibility

### DISCUSSION THREAD

**Discussion by abelsiqueira** [asreview/asreview#1403] [Ideas]
Following a discussion that we had this morning:

- What do we ask of users regarding citation (cite paper/cite asreview/cite asreview v1.1.1)?
- What do we ask of users regarding reproducibility (say which asreview version on paper/upload project folder to zenodo/upload requirements.txt and project folder)?
- Where do we tell them this?

One suggested answer for citation was to ask them to cite the paper and the software, possibly in different sections.
One suggested answer for reproducibility was to implement a way of generating an automatic equivalent to requirements.txt (i.e., the version of everything), showing the users how to upload these to OSF/zenodo, and giving commands to reproduce this using the uploaded material.
This all could also be shown on the "export" page and on the documentation.

One important comment is that the constraint to a specific version of asreview is not enough to ensure that other versions are the same.

**Comment by Rensvandeschoot** [comment]
related to https://github.com/asreview/asreview/issues/1402

**Comment by VeenDuco** [comment]
I'll go and try to find something for this issue to fix it.

**Comment by VeenDuco** [comment]
See #1545 for a first pull request.
